io-engine/src/grpc/v0/mayastor_grpc.rs

```diff
@@ -168,12 +169,27 @@ impl From<LvsError> for Status {
             LvsError::Destroy {
                 source, ..
             } => source.into(),
+
             LvsError::Invalid {
                 ..
             } => Status::invalid_argument(e.to_string()),
+
+            LvsError::PoolNotFound {
+                ..
+            } => Status::not_found(e.to_string()),
+
```

Comment:
unless you're grouping these, I'd remove the newlines between them

Reply:
Incorporated